packages/vuetify/src/components/VBreadcrumbs/VBreadcrumbs.tsx

```diff
@@ -111,12 +117,12 @@ export const VBreadcrumbs = genericComponent<new <T>() => {
             </VDefaultsProvider>
           ) }
 
-          { props.items.map((item, index, array) => (
+          { items.value.map((item, index, array) => (
             <>
               <VBreadcrumbsItem
-                key={ index }
+                key={ item.title }
                 disabled={ index >= array.length - 1 }
-                { ...(typeof item === 'string' ? { title: item } : item) }
+                { ...item }
                 v-slots={{
                   default: slots.title ? () => slots.title?.({ item, index }) : undefined,
```

Comment:
Technically a breaking change, item in the slot could have been a string before but now it's `{ title: string }`

Reply:
;(